internal/function/blame.go

```diff
@@ -0,0 +1,186 @@
+package function
+
+import (
+	"fmt"
```

Comment:
organize imports

Reply:
Fixed and configured Goland to run gofmt and goimports on every save.